examples/app_engine/versions.tf

```diff
@@ -18,12 +18,10 @@ terraform {
   required_version = ">= 0.13"
   required_providers {
```

Comment:
Are we keeping this so as not to break incase of null/random provider major updates?

Reply:
I only pulled the google, google-beta, let me expand to the rest in examples.

Reply:
IMO we could just remove versions.tf from examples

Reply:
Yeah - I will try that.

Reply:
Two examples use a provider that is not required by any modules, so need to keep those, but we can remove the rest.